.changeset/selfish-candles-relate.md

```diff
@@ -0,0 +1,6 @@
+---
+"@kaizen/component-library": minor
+"@kaizen/select": minor
```

Comment:
Why is select here?

Reply:
When I ran changeset it came up as a change, but looks like it might be a mistake. I've removed it now.